lang/solidity/Portfile

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           boost 1.0
+
+github.setup        ethereum solidity 0.8.19 v
+github.tarball_from downloads
+distname            ${github.project}_${github.version}
+
+revision            0
+
+name                solidity
+categories          lang
+platforms           darwin
```

Comment:
`platforms           darwin` is implicit, you can delete this line

Reply:
It looks good. thanks :)